src/storage.h

```diff
@@ -4,17 +4,25 @@
 #define SRC_STORAGE_H_
 
 #include <iostream>
+#include <vector>
+#include <string>
 
 #include <QtAlgorithms>
 #include <QDirIterator>
 #include <QFileInfo>
 #include <QVector>
 
+#include <theia/theia.h>
+
 #include <glog/logging.h>
+#include "options.h"
```

Comment:
adding a space before the include would be more consistent with the rest of code